components/WhyChooseUs/cardItem.styles.ts

```diff
@@ -0,0 +1,50 @@
+export const cartItemStyles = [
+  {
+    title: "Reason one",
+    titleStyles: [
+      "text-center",
+      "rounded-md",
+      "w-40",
+      "py-1.5",
+      "px-6",
+      "bg-cardBorderBlue",
+      "relative",
+      "top-4",
+    ].join(" "),
+    details: ["reason blablabla", "reason blablabl", "reason blablabla"],
+    containerStyles: ["w-full", "rounded-lg", "bg-template", "p-1"].join(" "),
+    gradient: ["bg-cardBorderBlue"],
+  },
+  {
+    title: "Reason two",
+    titleStyles: [
+      "text-center",
+      "rounded-md",
+      "w-40",
+      "py-1.5",
+      "px-6",
+      "bg-btn",
+      "relative",
+      "top-4",
+    ].join(" "),
+    details: ["reason blablabla", "reason blablabl", "reasonblablabla"],
+    containerStyles: ["w-full", "rounded-lg", "bg-btn", "p-1"].join(" "),
+    gradient: ["bg-btn"],
+  },
+  {
+    title: "Reason three",
+    titleStyles: [
+      "text-center",
+      "rounded-md",
+      "w-40",
+      "py-1.5",
+      "px-6",
+      "bg-cardBorder",
+      "relative",
+      "top-4",
+    ].join(" "),
+    details: ["reason blablabla", "reason blablabl", "reasonblablabla"],
+    containerStyles: ["w-full", "rounded-lg", "bg-cardBorder", "p-1"].join(" "),
+    gradient: ["bg-cardBorder"],
+  },
+];
```

Comment:
cartItemStyles were pretty good! I learned a new logic for the CSS from it! Good job!